alonzo/test/test/Test/Cardano/Ledger/Alonzo/Trials.hs

```diff
@@ -216,18 +262,26 @@ go n =
       ( localOption
           -- (QuickCheckReplay (Just 117392)) -- after 206 tests
           (QuickCheckReplay (Just n))
-          (testProperty "preserves ADA" $ adaPreservationChain @(AllegraEra TestCrypto))
-          -- (propertyTests  @(AlonzoEra TestCrypto))
+          -- (testProperty "preserves ADA" $ adaPreservationChain @(AllegraEra TestCrypto))
```

Comment:
did you mean to have this commented out?

Reply:
This is just a way to run a bunch of different tests easily. I will clean this up.

Reply:
cleaned up